app/views/container_image/guest_applications.html.haml

```diff
@@ -0,0 +1 @@
+= render :partial => 'guest_applications'
```

Comment:
@dkorn you don't need to add this partial instead you should handle the click to list of package in  container_image/show.html.haml similar to https://github.com/ManageIQ/manageiq/blob/master/app/views/host/show.html.haml#L7